src/Cake.Core/Tooling/ToolResolutionStrategy.cs

```diff
@@ -130,6 +136,9 @@ private FilePath LookInPath(string tool)
                     }
                 }
 
+                var allPaths = string.Join(",", _path);
+                _log.Verbose($"Could not resolve path for tool \"{tool}\" using these directories:");
```

Comment:
Could we combine the two log calls to a single one?

Reply:
Yes. I just felt it looked a bit nicer with them split up, bot no problem joining them into one statement.

Reply:
Hm, could it be `_log.Debug();`? I'm not seeing a `_log.Dianostic()`